### ISSUE TITLE
 The accuracy of the two figures does not match

### ISSUE THREAD

**Issue by cuicheng01** [apple/ml-cvnets#11]
Hi，thanks for such a great job，but I have a question, the accuracy of the standard training method of MobileViT-S in (b) and (c) in Figure 9 in the paper seems to be different, and the top-1 accuracy in (b) should be about 77% , the top-1 accuracy in (c) should be around 78%.
![image](https://user-images.githubusercontent.com/45199522/154027601-4f097c89-fe1c-4119-969b-242c09f10360.png)

Can anyone answer my doubts? 
thanks！

**Comment by sacmehta** [comment]
Thanks for your question. The validation error plot in (b) is without EMA while the validation accuracy in (c) is with EMA.

**Comment by cuicheng01** [comment]
Thanks a lot for the reply. It is surprising that the EMA strategy can rise so much. My previous experimental conclusions are basically between 0-0.2%. 
By the way, the top-1 accuracy of the standard training method is only 77% without using the EMA strategy?

**Comment by sacmehta** [comment]
Our experiments also suggests that EMA improves performance by 0.2-0.3%.

The top-1 accuracy without EMA is about 78%. The difference we observe  in graph is because part (c) performance is measured after the training on a single GPU ensuring batches are neither truncated nor padded. While the data for plots in (b) are measured during training. For (b), depending on the batch size, there could be cases when dataset size is not multiple of batch size. For example, for ImageNet, we have 50k validation images. If we use a batch size of 1024 for computing validation statistics during training, then few batches will be either truncated or padded. So, training/validation curves could be bit noisy.

Hope this helps.

**Comment by cuicheng01** [comment]
ok！Thanks for your reply. However, I still have some questions. According to what you said, I reproduced the process. Although there will be some differences in accuracy, it will not be as big as 1%. I would like to take the liberty to ask, can you provide the training log here?
I look forward to your reply

**Comment by sacmehta** [comment]
Hi @cuicheng01 ,

Extremely sorry for the late response. I did not realize that I did not respond to your comment.

Here are the training/validation/validation + EMA loss curves. Hope this helps.

Train loss vs. Epoch:
![Train_Loss](https://user-images.githubusercontent.com/18603245/176094462-4c4c9b3f-7cb9-4562-8240-095a38004793.svg)

Val loss vs. Epoch:
![Val_Loss](https://user-images.githubusercontent.com/18603245/176094486-c52e8c64-8662-4025-a897-335458b30537.svg)

Val loss w/ EMA vs. Epoch:
![Val_EMA_Loss](https://user-images.githubusercontent.com/18603245/176094495-0cd41934-ce33-4789-a8e6-a34f9f0a5d40.svg)

**Comment by sacmehta** [comment]
Closing issue because of no activity.
